libs/tree-urls-client/src/tree_urls_client/core.clj

```diff
@@ -1,14 +1,26 @@
-(ns terrain.clients.tree-urls
-  (:use [terrain.util.config]
-        [clojure-commons.error-codes]
+(ns tree-urls-client.core
```

Comment:
This is just copied from terrain for the most part, obviously. It's possible at some point we should switch to using `swagger-codegen` for projects which have swagger docs (and maybe maintain swagger.yaml files for the ones that don't? not sure), but the interfaces the generated clojure clients expose are somewhat different than what we've generally been doing, and tree-urls doesn't have swagger anyway.